policyengine_us/variables/gov/states/co/tax/income/subtractions/pension/co_pension_subtraction_indv.py

```diff
@@ -0,0 +1,60 @@
+from policyengine_us.model_api import *
+
+
+class co_pension_subtraction_indv(Variable):
+    value_type = float
+    entity = Person
+    label = "Colorado pension and annuity subtraction for eligible individuals"
+    defined_for = StateCode.CO
+    unit = USD
+    reference = (
+        "https://tax.colorado.gov/sites/tax/files/documents/DR0104AD_2022.pdf#page=1",
+        "https://tax.colorado.gov/sites/tax/files/documents/DR_104_Book_2022.pdf#page=12",
+        "https://law.justia.com/codes/colorado/2022/title-39/article-22/part-1/section-39-22-104/"
+        # C.R.S. 39-22-104(4)(g)(III)
+    )
+    definition_period = YEAR
+
+    def formula(person, period, parameters):
+        p = parameters(period).gov.states.co.tax.income.subtractions.pension
+        taxable_pension_income = person("taxable_pension_income", period)
+        pension_survivors = person("pension_survivors", period)
+        co_social_security_subtraction = person(
+            "co_social_security_subtraction_indv", period
+        )
+        age = person("age", period)
+        head = person("is_tax_unit_head", period)
+        spouse = person("is_tax_unit_spouse", period)
+        head_or_spouse = head | spouse
+        age_head_or_spouse = age * head_or_spouse
```

Comment:
Why set age to zero for dependents? If this concerns eligibility I think you want to represent it differently.

Reply:
The [tax form](https://tax.colorado.gov/sites/tax/files/documents/DR_104_Book_2022.pdf#page=12) limits the pension subtraction to only the head and spouse. What would be a cleaner way of doing it?